settings.gradle.kts

```diff
@@ -17,7 +17,7 @@
 rootProject.name = "knotx-stack"
 
 if (file(".composite-enabled").exists()) {
-    includeBuild("../knotx-gradle-plugins")
+//    includeBuild("../knotx-gradle-plugins")
```

Comment:
removing better then commenting :)

Reply:
Sure, currently I left that for dev purposes.